.devcontainer/devcontainer.json

```diff
@@ -25,7 +26,9 @@
     "vscode": {
       // Set *default* container specific settings.json values on container create.
       "settings": {
-        "python.defaultInterpreterPath": "/usr/local/bin/python"
+        "python.defaultInterpreterPath": "/usr/local/bin/python",
+        "python.testing.pytestEnabled": true,
+        "python.testing.unittestEnabled": false
```

Comment:
```
[ERROR 2023-4-3 2:40:29.20]: Error discovering pytest tests:
 [n [Error]: Setting random seed to 42
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/usr/local/lib/python3.10/shutil.py", line 679, in _rmtree_safe_fd
INTERNALERROR>     os.unlink(entry.name, dir_fd=topfd)
INTERNALERROR> PermissionError: [Errno 13] Permission denied: 'stepwise'
INTERNALERROR> 
INTERNALERROR> During handling of the above exception, another exception occurred:
INTERNALERROR> 
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/main.py", line 265, in wrap_session
INTERNALERROR>     config._do_configure()
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1046, in _do_configure
INTERNALERROR>     self.hook.pytest_configure.call_historic(kwargs=dict(config=self))
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/pluggy/_hooks.py", line 277, in call_historic
INTERNALERROR>     res = self._hookexec(self.name, self.get_hookimpls(), kwargs, False)
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/pluggy/_manager.py", line 80, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/pluggy/_callers.py", line 60, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/pluggy/_result.py", line 60, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/pluggy/_callers.py", line 39, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/cacheprovider.py", line 504, in pytest_configure
INTERNALERROR>     config.cache = Cache.for_config(config, _ispytest=True)
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/cacheprovider.py", line 83, in for_config
INTERNALERROR>     cls.clear_cache(cachedir, _ispytest=True)
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/cacheprovider.py", line 96, in clear_cache
INTERNALERROR>     rm_rf(d)
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/pathlib.py", line 170, in rm_rf
INTERNALERROR>     shutil.rmtree(str(path), onerror=onerror)
INTERNALERROR>   File "/usr/local/lib/python3.10/shutil.py", line 725, in rmtree
INTERNALERROR>     _rmtree_safe_fd(fd, path, onerror)
INTERNALERROR>   File "/usr/local/lib/python3.10/shutil.py", line 658, in _rmtree_safe_fd
INTERNALERROR>     _rmtree_safe_fd(dirfd, fullname, onerror)
INTERNALERROR>   File "/usr/local/lib/python3.10/shutil.py", line 681, in _rmtree_safe_fd
INTERNALERROR>     onerror(os.unlink, fullname, sys.exc_info())
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/pathlib.py", line 122, in on_rm_rf_error
INTERNALERROR>     chmod_rw(str(parent))
INTERNALERROR>   File "/home/vscode/.local/lib/python3.10/site-packages/_pytest/pathlib.py", line 115, in chmod_rw
INTERNALERROR>     os.chmod(p, mode | stat.S_IRUSR | stat.S_IWUSR)
INTERNALERROR> PermissionError: [Errno 1] Operation not permitted: '/workspace/Auto-GPT/.pytest_cache/v/cache'

Traceback (most recent call last):
  File "/home/vscode/.vscode-server/extensions/ms-python.python-2023.6.1/pythonFiles/testing_tools/run_adapter.py", line 22, in <module>
    main(tool, cmd, subargs, toolargs)
  File "/home/vscode/.vscode-server/extensions/ms-python.python-2023.6.1/pythonFiles/testing_tools/adapter/__main__.py", line 99, in main
    parents, result = run(toolargs, **subargs)
  File "/home/vscode/.vscode-server/extensions/ms-python.python-2023.6.1/pythonFiles/testing_tools/adapter/pytest/_discovery.py", line 47, in discover
    raise Exception("pytest discovery failed (exit code {})".format(ec))
Exception: pytest discovery failed (exit code 3)

	at ChildProcess.<anonymous> (/home/vscode/.vscode-server/extensions/ms-python.python-2023.6.1/out/client/extension.js:2:238829)
	at Object.onceWrapper (node:events:646:26)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)]

```
Got this error when clicking on the flask in the sidebar

Reply:
I'll include the installation of the requirements in a separate PR. I'm not able to push directly to desojo:master

Reply:
I'll include the installation of the requirements in a separate PR. I'm not able to push directly to desojo:master